tests/test.cpp

```diff
@@ -481,7 +483,7 @@ std::map<std::string, std::function<void()>> construct_tests() {
 
    EXPECT(y.defined());
    EXPECT(x.sizes().vec() == y.sizes().vec());
-   EXPECT(x.eq(y).all());
+   EXPECT(x.eq(y).all().toCInt());
```

Comment:
I wonder if
bool Tensor::allclose()
is not more readable/fitting in those comparisons?

Reply:
Yes, good find! This didn't exist when I last updated PyTorch.